README.md

```diff
@@ -1,29 +1,24 @@
 <!-- markdownlint-disable MD033 -->
-# Plug and Play BugBash instructions for May2020 release
+# Plug and Play BugBash instructions for the Summer Refresh 2020 release
 
 ## Intro
 
-This internal bug bash explores the new IoT Hub features to support the Plug and Play May 2020 release. Using a  DTDL v2 Model to describe a device with one  sensor interface that sends telemetry, accepts commands and reports properties. The sample is based on a device simulator connected to IoT Hub with a client SDK and can be managed using the service SDKs  to read telemetry and interact with properties and commands.
+This bug bash is focussed on the new IoT Hub features, SDKs, and tooling for the Plug and Play Summer Refresh 2020 release. There are two options during this bug bash. 
 
-Review the docs in the private staging environment [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=pr-en-us-114965)
+Option 1 is to follow Quickstarts in the Plug and Play documentation to build the samples provided there and run them against the IoT Explorer, a tool we're providing with this release. 
 
-The [DTDL v2 Spec](https://microsoft.sharepoint.com/:w:/t/Azure_IoT/IoTPlat/EaotICYXiQVOh_xnBggJwqMBMKGoOyMzV1qFYsOdcyvOVw?e=VyG0QN) can be used as a reference for the language. An easiest way to get started with the language is by looking at the [sample models](./models) created for this Bug Bash.
+Option 2 is to use a set of intructions we're providing to build a simulated Plug and Play device and a solution to validate the workflow end to end.
```

Comment:
I think Option 2 should also use the docs, we are providing Stefan's deck just as a workaround but ideally it should not be required and docs should be enough. 

**docs should provide instructions to build pnp solutions**

I would leave Option 2 for the second BugBash.

Reply:
Unless there is a downside to it, i will leave Option 2 given this is going to be continuous 2 week bug bash that folks can participate at their convenience.